src/datasets/containers.jl

```diff
@@ -50,13 +50,14 @@ struct TableDataset{T}
     TableDataset{T}(table::T) where T = Tables.istable(table) ? new{T}(table) : error("Object doesn't implement Tables.jl interface")
 end
 
+TableDataset(table::T) where {T} = TableDataset{T}(table)
 TableDataset(path::AbstractPath) = TableDataset(DataFrame(CSV.File(path)))
 
 function LearnBase.getobs(dataset::FastAI.Datasets.TableDataset{T}, idx) where {T}
     if Tables.rowaccess(dataset.table)
         for (index, row) in enumerate(Tables.rows(dataset.table))
```

Comment:
It's not faster, but instead of a loop + index check you could write something like `first(Iterators.drop(Tables.rows(...), idx - 1))` (skip up until the index and return the next element, which will be the element at the index).

Reply:
Yeah, I can do that.